docs/arch/agent.md

```diff
@@ -0,0 +1,50 @@
+# Summary
+
+The agent is the powerhouse of SealCi. It receives actions and runs them in order to complete the operational part of the CI.
+
+# References
+
+## Actions
+
+A action is a defined as list of commands. actions are launched in a controlled environment, as defined by the agent configuration.
```

Comment:
```suggestion
An action is defined as a list of commands. Actions are launched in a controlled environment, as defined by the agent configuration.
```

Reply:
The agent configuration does not defined the actions execution environment. The pipeline definition describes each action execution envronment. See https://github.com/dev-sys-do/sealci/pull/4